schemas/protocol.fbs

```diff
@@ -127,4 +127,7 @@ table Inv {
     inventory: [InvVector] (required);
 }
 
+table GetData {
+    inventory: [InvElem] (required);
```

Comment:
This should be `InvVector` I think, I don't even know how this compiles as `InvElem` is not on your schema. Please fix this and then recompile to get a new `protocol_generated.rs`
```suggestion
    inventory: [InvVector] (required);
```